streaming/text/convert/enwiki/tfrecord/make_train_parallel.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+num_cpus = os.cpu_count()
+num_shards = 500
+input_file_pattern = '/tmp/enwiki_preproc/results4/part-%05d-of-00500'
```

Comment:
Does this path exist ? Should we get it from command line as argparse ?

Reply:
Done